site-src/implementations.md

```diff
@@ -0,0 +1,95 @@
+| [v0.2.x][v0.2] | :heavy_check_mark: | :heavy_plus_sign:  | :heavy_plus_sign:  |:heavy_plus_sign:  | :heavy_minus_sign: | :heavy_plus_sign:  |
+
+- :heavy_check_mark: available
+- :heavy_plus_sign: in progress
+- :heavy_minus_sign: not planned
+
+[1]:#contour
+[2]:#google-cloud-platform
+[3]:#istio
+[4]:#kong
+[5]:#knative
+[6]:#traefik
+
+[v0.1]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.1.0
+[v0.2]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.2.0
+
+## Project References
+
+In this section you will find specific links to blog posts, documentation and other Gateway API references for specific implementations.
+
+### Contour
+
+[Contour][contour] is an open source ingress controller for Kubernetes.
+
+Contour currently implements the latest `v0.2.x` Gateway API Specification.
+
+See the [Contour Gateway API Guide][contour-1] for information on how to deploy and use Contours's Gateway implementation.
+
+[contour]:https://projectcontour.io
+[latest]:https://doc.traefik.io/traefik/routing/providers/kubernetes-gateway/
+[contour-1]:https://projectcontour.io/guides/gateway-api/
+
+### Google Cloud Platform
+
+The [Google Cloud Platform (GCP)][gcp] is a cloud computing platform and infrastructure provider.
+
+GCP is actively working towards an implementation of the Gateway API `v0.2.x` specification, and status updates and documentation will be provided here as the work progresses.
+
+[gcp]:https://cloud.google.com
+
+### Istio
+
+[Istio][istio] is an open source [service mesh][mesh] for Kubernetes clusters.
+
+Istio currently supports the previous `v0.1.x` Gateway API specification and is working towards supporting the latest `v0.2.x` release.
+
+Status updates and documentation for `v0.2.x` will be provided here as the work progresses.
+
+See the [Istio Gateway API Documentation][istio-1] for information on how to deploy and use Istio's Gateway implementation.
+
+[istio]:https://istio.io
+[mesh]:https://istio.io/latest/docs/concepts/what-is-istio/#what-is-a-service-mesh
+[istio-1]:https://istio.io/latest/docs/tasks/traffic-management/ingress/service-apis/
+
+### Kong
+
+[Kong][kong] is an open source API Gateway built for hybrid and multi-cloud environments.
+
+Kong is actively working towards an implementation of the Gateway API `v0.2.x` specification for it's [Kubernetes Ingress Controller][kic] and status updates and documentation will be provided here as the work progresses.
+
+You can subscribe to [kong/kubernetes-ingress-controller/issues/692][kong-1] to track the implementation progress and contribute.
+
+[kong]:https://konghq.com
+[kic]:https://github.com/kong/kubernetes-ingress-controller
+[kong-1]:https://github.com/kong/kubernetes-ingress-controller/issues/692
+
+### Knative
+
+[Knative][knative] is an open source Kubernetes-based platform to deploy and manage modern serverless workloads.
```

Comment:
TODO: again need to make sure the Knative maintainers are on board with this representation. Also I'm _not sure_ whether they're actively working on an implementation or planning to. I'll do more digging before taking this out of draft state.

Reply:
@nak3 Do you have any insight you can provide here?

Reply:
Yes, It is a little bit weird that Knative is listed in the Implementation. Knative is going to support the Ingress which implemented gateway-api, so Knative is a kind of "user" rather than "implementation" as John Howard also mentioned below.

Reply:
Ok, for now I've opted to remove it from the doc for now and then we can add further context as needed in other iterations: 7781c9acbc944c0c05270bc322bd2ba24509a17c